Library/Homebrew/cmd/tests.rb

```diff
@@ -1,9 +1,34 @@
 require "fileutils"
+require "official_taps"
 
 module Homebrew
   def tests
+    ENV["HOMEBREW_NO_ANALYTICS_THIS_RUN"] = "1"
+    Homebrew.install_gem_setup_path! "bundler"
+
+    if ARGV.include? "--official-cmd-taps"
+      OFFICIAL_CMD_TAPS.each do |org, repo|
+        tap = "#{org}/#{repo}"
+        tap_dir = HOMEBREW_LIBRARY/"Taps/#{org}/homebrew-#{repo}"
+
+        # It has no tests :(
+        next if "#{tap}" == "homebrew/services"
+        ohai "Running #{tap} tests:"
+
+        system "brew", "tap", tap unless tap_dir.exist?
+        tap_dir.cd do
+          unless quiet_system("bundle", "check")
+            system "bundle", "install", "--path", "vendor/bundle"
```

Comment:
If we're not committing: I don't think it matters but probably not a bad idea anyway.